From d4b9c22c1f8465bda5b6a83dc7e2cf59c3fe44e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Wed, 27 Jan 2016 08:54:31 +0100
Subject: [PATCH] Enable localization before redirect
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This is needed in case of IIS which needs full HTML response.

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 libraries/common.inc.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libraries/common.inc.php b/libraries/common.inc.php
index fd85814f943..773e8a32001 100644
--- a/libraries/common.inc.php
+++ b/libraries/common.inc.php
@@ -340,6 +340,7 @@
 if ($GLOBALS['PMA_Config']->get('ForceSSL')
     && ! $GLOBALS['PMA_Config']->detectHttps()
 ) {
+    require './libraries/select_lang.lib.php';
     // grab SSL URL
     $url = $GLOBALS['PMA_Config']->getSSLUri();
     // Actually redirect
